src/staker.cairo

```diff
@@ -96,6 +107,13 @@ pub mod Staker {
         }
     }
 
+    #[derive(Drop, Serde, starknet::Store)]
+    struct StakingLogRecord {
+        timestamp: u64,
+        total_staked: u128,
+        cumulative_seconds_per_total_staked: UFixedPoint,
+    }
```

Comment:
todo: store pack the timestamp and total staked into one felt252

Reply:
My bad. Thank you for catching that. I should've used felt252 to store in a single slot.